module/Application/src/Module.php

```diff
@@ -14,6 +14,6 @@ class Module
 {
     public function getConfig() : array
     {
-        return include __DIR__ . '/../config/module.config.php';
+        return (array) include __DIR__ . '/../config/module.config.php';
```

Comment:
I think we can do with : 

```php
/** @var array $config */
$config = include __DIR__ . '/../config/module.config.php';
return $config;
```

instead of `(array)` cast as the `config/module.config.php` is an array already.